test/routes/forms/submission/router.test.ts

```diff
@@ -0,0 +1,42 @@
+import { vi, describe, it, expect, beforeAll } from "vitest";
+import request from "supertest";
+import express from "express";
+import router from "../../../../src/routes/forms/router";
+
+vi.mock("../../../../src/middleware/authentication/middleware", () => ({
+  default: (_req, _res, next) => next(),
+}));
+
+vi.mock("../../../../src/middleware/rateLimiter/middleware", () => ({
+  default: (_req, _res, next) => next(),
+}));
```

Comment:
This is my quick/dirty solution to preventing middleware from blocking the router unit tests.  I'm sure there's a better way.